proposals/incubation/kyverno.md

```diff
@@ -0,0 +1,223 @@
+- Apache 2.0 github.com/google/go-containerregistry 
+- Apache 2.0 github.com/google/go-containerregistry/pkg/authn/kubernetes 
+- Apache 2.0 github.com/googleapis/gnostic 
+- Apache 2.0 github.com/in-toto/in-toto-golang 
+- Apache 2.0 github.com/jmespath/go-jmespath 
+- Apache 2.0 github.com/kyverno/go-wildcard
+- Apache 2.0 github.com/lensesio/tableprinter 
+- Apache 2.0 github.com/mattbaird/jsonpatch 
+- Apache 2.0 github.com/prometheus/client_golang 
+- Apache 2.0 github.com/sigstore/cosign 
+- Apache 2.0 github.com/sigstore/sigstore 
+- Apache 2.0 github.com/spf13/cobra 
+- Apache 2.0 gopkg.in/yaml.v2 
+- Apache 2.0 gotest.tools 
+- Apache 2.0 k8s.io/api 
+- Apache 2.0 k8s.io/apiextensions-apiserver 
+- Apache 2.0 k8s.io/apimachinery 
+- Apache 2.0 k8s.io/cli-runtime 
+- Apache 2.0 k8s.io/client-go 
+- Apache 2.0 k8s.io/klog/v2 
+- Apache 2.0 k8s.io/kube-openapi 
+- Apache 2.0 sigs.k8s.io/controller-runtime 
+- Apache 2.0 sigs.k8s.io/kustomize/api 
+- Apache 2.0 sigs.k8s.io/kustomize/kyaml 
+- BSD 3-Clause github.com/evanphx/json-patch 
+- BSD 3-Clause github.com/evanphx/json-patch/v5 
+- BSD 3-Clause github.com/julienschmidt/httprouter
+- BSD 2-Clause github.com/pkg/errors 
+- BSD-3-Clause gopkg.in/inf.v0 
+- MIT github.com/aquilax/truncate 
+- MIT github.com/blang/semver/v4 
+- MIT github.com/cenkalti/backoff 
+- MIT github.com/fatih/color 
+- MIT github.com/ghodss/yaml 
+- MIT github.com/kataras/tablewriter
+- MIT github.com/onsi/ginkgo 
+- MIT github.com/onsi/gomega 
+- MIT github.com/orcaman/concurrent-map 
+- MIT github.com/patrickmn/go-cache 
+- MIT github.com/robfig/cron/v3 
+- MIT github.com/stretchr/testify 
+- MIT and Apache 2.0 gopkg.in/yaml.v3 
+- MIT sigs.k8s.io/yaml 
+
+
+### Community Stats (graphs / dashboards) 
+The Kyverno DevStats dashboards can be found [here](https://kyverno.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=15m&viewPanel=4).
+
+We are using the following metrics as key indicators of community health:
+
+- [Stargazers](https://kyverno.devstats.cncf.io/d/81/community-health?orgId=1&var-repo_name=All&var-metric=Stargazers&var-table=swatchers&var-pref=all&var-met1=watch&var-met2=watch&from=now-1y&to=now): 372% growth from 524 -> 1949.
+- [Issue Creators](https://kyverno.devstats.cncf.io/d/81/community-health?orgId=1&var-repo_name=All&var-metric=Issue%20creators&var-table=scommunity_health&var-pref=&var-met1=chealthissue&var-met2=&from=now-1y&to=now): 602% growth from 45 -> 271.
+- [Code Committers](https://kyverno.devstats.cncf.io/d/81/community-health?orgId=1&var-repo_name=All&var-metric=Code%20committers&var-table=scommunity_health&var-pref=&var-met1=chealthcommit&var-met2=&from=now-1y&to=now): 594% growth from 18 -> 107.
+- [Code Commenters](https://kyverno.devstats.cncf.io/d/81/community-health?orgId=1&var-repo_name=All&var-metric=Code%20commenters&var-table=scommunity_health&var-pref=&var-met1=chealthcomment&var-met2=&from=now-1y&to=now): 573% growth from 22 -> 126.
```

Comment:
Should we update all of these numbers to the latest or pin it to a specific date?

For eg a pinned grafana url https://kyverno.devstats.cncf.io/d/81/community-health?orgId=1&var-repo_name=All&var-metric=Stargazers&var-table=swatchers&var-pref=all&var-met1=watch&var-met2=watch&from=1619425267588&to=1650961267588

Reply:
Updated!